src/Microsoft.Identity.Client/AppConfig/ApplicationOptions.cs

```diff
@@ -78,6 +116,8 @@ public abstract class ApplicationOptions
         public string SliceParameters { get; set; }
 
         /// <summary>
+        /// Identifier of the component (libraries/SDK) consuming MSAL.NET.
+        /// This will allow for disambiguation between MSAL usage by the app vs MSAL usage by component libraries.
         ///     TODO: do we have a better / more descriptive name for this?
         /// </summary>
         public string Component { get; set; }
```

Comment:
I am not familiar with this concept and don't understand what it is / how it helps the developers. Is it for our telemetry ?

Reply:
I agree with @bogavril

---
In reply to: [248333026](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/790#discussion_r248333026) [](ancestors = 248333026)

Reply:
It's for us (our telemetry). this is mainly 1P